vscode/src/context/context-filters-provider.ts

```diff
@@ -120,10 +141,16 @@ function checkFilter(
 }
 
 function parseContextFilterItem(item: CodyContextFilterItem): ParsedContextFilterItem {
-    const repoNamePattern = new RE2(item.repoNamePattern)
+    const repoNamePattern = new RE2(item.repoNamePattern, 'u')
     const filePathPatterns = item.filePathPatterns
-        ? item.filePathPatterns.map(pattern => new RE2(pattern))
+        ? item.filePathPatterns.map(pattern => new RE2(pattern, 'u'))
         : undefined
 
     return { repoNamePattern, filePathPatterns }
 }
+
+/**
+ * A a singleton instance of the `ContextFiltersProvider` class.
+ * `contextFiltersProvider.init` should be called on extension activation.
```

Comment:
```suggestion
 * `contextFiltersProvider.init` should be called and awaited on extension activation.
```

I suppose we also need to await it so we know the cache is properly seeded and we don't start requests before that's the case? 🦭